beets/dbcore/query.py

```diff
@@ -20,6 +20,14 @@
 from datetime import datetime, timedelta
 
 
+class InvalidQuery(ValueError):
```

Comment:
Let's call this `InvalidQueryError`. It's a [Python convention](https://www.python.org/dev/peps/pep-0008/#exception-names) to name most exceptions with that suffix.